### ISSUE TITLE
Create Blazor version of Razor Pages with Entity Framework Core 

### ISSUE THREAD

**Issue by Rick-Anderson** [dotnet/AspNetCore.Docs#33278]
### Description

Do this after #26505 closes
Related issue: 

### Page URL

https://learn.microsoft.com/en-us/aspnet/core/data/ef-rp/intro?view=aspnetcore-8.0&tabs=visual-studio

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/data/ef-rp/intro.md

### Document ID

fa488196-1f98-08ea-adcc-17d9dac86727

### Article author

@tdykstra

**Comment by guardrex** [comment]
Already covered by https://github.com/dotnet/AspNetCore.Docs/issues/32359.
